packages/gestalt/src/utils/positioningUtils.js

```diff
@@ -44,7 +44,7 @@ export const getContainerNode = ({
   // append the portal
   let containerNode = null;
   // currentNode references the DOM node used while traversing up nodes in the DOM tree
-  let currentNode = initialPositionRef;
+  let currentNode: ?HTMLElement | Node = initialPositionRef;
```

Comment:
On line 41 we're establishing that `initialPositionRef` is `?HTMLElement`, but on line 60 we're potentially re-assigning `currentNode` to the parent node of `currentNode`, which is of type `Node`